functions/src/on-create-bet.ts

```diff
@@ -305,3 +307,23 @@ const notifyFills = async (
 const currentDateBettingStreakResetTime = () => {
   return new Date().setUTCHours(BETTING_STREAK_RESET_HOUR, 0, 0, 0)
 }
+
+const updateContractMetrics = async (contract: Contract, user: User) => {
+  const betSnap = await firestore
+    .collection(`contracts/${contract.id}/bets`)
+    .where('userId', '==', user.id)
+    .get()
```

Comment:
Does this include the current bet that triggered the function? It does, doesn't it?

Reply:
I think it must, too!